app/models/user.rb

```diff
@@ -37,4 +38,11 @@ class User < ApplicationRecord
   def opted_in?
     providers.any?(&:opted_in?)
   end
+
+  def admin?
+    return true if email.end_with? "@education.gov.uk"
```

Comment:
You could condense this method to:

```suggestion
    email.end_with? "@education.gov.uk" || email.end_with? "@digital.education.gov.uk"
```

Reply:
or `return true if email.include?("education.gov.uk")`

Reply:
maybe not include

education.gov.uk@somewhere.com